app/views/gov_one/info.html.slim

```diff
@@ -0,0 +1,14 @@
+= render 'learning/cms_debug'
+
+- content_for :page_title do
+  = html_title t('gov_one.title')
+
+.govuk-grid-row
+  .govuk-grid-column-one-half
+    = m('gov-one-info.hero-body')
```

Comment:
This is wrong again, there were locales before or am I mistaken. You don't want to be creating `resources` in the `test` env.